.github/workflows/CI.yml

```diff
@@ -28,75 +28,51 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-
       - uses: dtolnay/rust-toolchain@stable
         with:
           components: rustfmt
-
       - run: cargo fmt --all --check
 
   test:
     name: Test ${{ matrix.rust }} on ${{ matrix.os }}
     needs: [style]
     strategy:
       matrix:
-        rust:
-          - stable
-          - beta
-          - nightly
-
-        os:
-          - ubuntu-latest
-          - windows-latest
-          - macOS-latest
```

Comment:
Can we keep all these bullet points? I find it easier to scan the matrix with my eyes that way.